Dockerfile.assisted-installer-deployment

```diff
@@ -7,6 +7,7 @@ RUN dnf install -y \
     make \
     skopeo \
     libffi-devel \
+    krb5-devel \
```

Comment:
This was a requirement from pip-compile, not sure why though

Reply:
what do you see if it's not present?

Reply:
> what do you see if it's not present?

Error when executing pip-compile (executed by make pip-freeze): `krb5-config: command not found`

Reply:
I think it comes from pip-compile, but I was also trying to add another unrelated dependency. I'll make sure it comes from pip-compile

Reply:
whoops, it was indeed coming from the other dependency. Mystery solved :sweat_smile: